subscriber.go

```diff
@@ -168,43 +98,91 @@ func (s *Subscriber) Receive() <-chan *Update {
 // HistoryDispatched must be called when all messages coming from the history have been dispatched.
 func (s *Subscriber) HistoryDispatched(responseLastEventID string) {
 	s.responseLastEventID <- responseLastEventID
-	close(s.history.in)
 }
 
 // Disconnect disconnects the subscriber.
 func (s *Subscriber) Disconnect() {
 	s.disconnectedOnce.Do(func() {
 		close(s.disconnected)
+		close(s.out)
```

Comment:
This change creates a deadlock. https://github.com/dunglas/mercure/pull/578/files#diff-802f9d2c1e1a0efff7a8474036a024fbbba988f5bdf4650a0b3b4dad99df9ca2R74 can be called after this, and block forever.